test/Sema/moveonly_restrictions.swift

```diff
@@ -44,12 +50,17 @@ struct SCopyable {
 
 struct S { // expected-note {{consider adding '~Copyable' to struct 'S'}}
     var copyable: CopyableKlass
-    var moveOnly2: MoveOnlyStruct? // expected-error {{type 'MoveOnlyStruct' does not conform to protocol 'Copyable'}}
-    var moveOnly: MoveOnlyStruct // expected-error {{stored property 'moveOnly' of 'Copyable'-conforming struct 'S' has non-Copyable type 'MoveOnlyStruct'}}
+    var moveOnly2: MoveOnlyStruct? // expected-error {{stored property 'moveOnly2' of 'Copyable'-conforming struct 'S' has non-Copyable type 'MoveOnlyStruct?'}}
+    // FIXME(NCG): Shouldn't this be also an error?
+    var moveOnly: MoveOnlyStruct
```

Comment:
I think we just diagnose one property and stop. I guess we could diagnose all of the noncopyable properties preventing conformance 🤷🏼